packages/taquito/src/injector/rpc-injector.ts

```diff
@@ -1,9 +1,23 @@
 import { Injector } from './interface';
 import { Context } from '../context';
+import { formatErrorMessage } from './helper';
+import { HttpResponseError } from '@taquito/http-utils';
 
 export class RpcInjector implements Injector {
   constructor(private context: Context) {}
-  inject(signedOperationBytes: string): Promise<string> {
-    return this.context.rpc.injectOperation(signedOperationBytes);
+  async inject(signedOperationBytes: string): Promise<string> {
+    let hash: string;
+    try {
+      hash = await this.context.rpc.injectOperation(signedOperationBytes);
+    } catch (error) {
+      const stringToStrip =
+        '. You may want to use --replace to provide adequate fee and replace it';
```

Comment:
do we have a method to generalize this string instead of checking for the raw value of it?